### DISCUSSION TITLE
Create "try until clothing fits" option for sewing kit and tailor's kit.

### DISCUSSION THREAD

**Discussion by metalheimer** [cataclysmbn/Cataclysm-BN#2808] [General]
### Is your feature request related to a problem? Please describe.

Those two kits have choices for "repair to full condition" and "repair until reinforced" but I think the missing step there is "repair until it fits". I, for one, am often aware when my char is not skilled enough to reinforce the item, and thus would like to stop after the fitted stage. The option "try until success or failure" is not suitable (one could even say it doesn't.... fit, hehe) and can result in failure, possibly several times, thus interrupting the process... several times. Having the "until fit" option would smooth (smoothen? ensmooth?) the process.

### Describe the solution you'd like

I bled my heart and soul above and you ask for more text? No, you vampire form. No.

**Comment by chaosvolt** [comment]
Could work, though personally I prefer the "try until success" option because if your skill is low enough to risk causing a failure spiral, using any of the "just keep going" options will likely destroy the item or render it unusably damaged before you notice.

**Comment by metalheimer** [comment]
>  ..will likely destroy the item or render it unusably damaged before you notice.

Gives me an idea that there could be a pause and a prompt like "Next failure will destroy the item. Continue?"

**Comment by chaosvolt** [comment]
Could be a good idea, yeah. Warning once success rate drops to 0% is also an option, though damage spirals can likely occur anytime you get to the point where failure chance is too close to success chance.
